src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -2364,7 +2369,7 @@ class RandomCrop(tvt_v2.Transform, NumpytoTVTensorMixin):
 
     def __init__(
         self,
-        crop_size: tuple,  # (H, W)
+        crop_size: tuple[int, int],  # (H, W)
```

Comment:
Is this always true in OTX? No (W, H)? OK if it is so but we'd better modify test cases for non-square input size to check for the future changes.

Reply:
I just updated type hint for the input size feature, so I don't know well this. But when I checked the code, crop_size should be (H, W) because this class refers `ImageInfo` class and it has a tuple w/ (H, W) order. I'll update the unit test to use non-square value.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3759/commits/2385183a05f5e4bc871e2c461287abcbf344008c